homeassistant/components/husqvarna_automower/coordinator.py

```diff
@@ -64,24 +68,27 @@ async def client_listen(
         hass: HomeAssistant,
         entry: ConfigEntry,
         automower_client: AutomowerSession,
-        reconnect_time: int = 2,
     ) -> None:
         """Listen with the client."""
         try:
             await automower_client.auth.websocket_connect()
-            reconnect_time = 2
-            await automower_client.start_listening()
+            await automower_client.start_listening()  # Ensure we catch errors here too
+            self.reconnect_time = DEFAULT_RECONNECT_TIME  # Reset reconnect time after successful connection
```

Comment:
Move the comment above the code line so the lines are kept within 88 characters.

Reply:
done in https://github.com/home-assistant/core/pull/123085/commits/40632e3aea0b48caab264c9c461f03f1994cfffc